optimizely/optimizely.py

```diff
@@ -1369,25 +1369,29 @@ def _fetch_qualified_segments(self, user_id: str, options: Optional[list[str]] =
     def send_odp_event(
         self,
         action: str,
+        identifiers: dict[str, str],
```

Comment:
Is there a reason we moved identifiers higher in the arg list?
I believe it should be in the third place? I've tested without keyword args where order mattered ...
![image](https://user-images.githubusercontent.com/1386553/221332089-5a0707a4-2d4d-4a1a-9a7c-d8c2240bc9cc.png)

Reply:
Same reason why `action` is first instead of `type`- parameters with defaults must come after non-default parameters.